### ISSUE TITLE
Metric Hyperparameters investigation/sanity check

### ISSUE THREAD

**Issue by eddableheath** [alan-turing-institute/ARC-LoCoMoSeT#36]
With the strange behaviour of PARC when evaluating ImageNet-1k we wish to evaluate the hyperparameters used to see these are the source of the issue. This method can be generalised away from PARC however, with sanity checks for the hyperparameters of the other metrics looking to be worthwhile.

**Comment by eddableheath** [comment]
[37](https://github.com/alan-turing-institute/ARC-LoCoMoSeT/issues/37) should be done before this to recreate the method properly.

**Comment by jack89roberts** [comment]
This should be independent from #37 I think (btw if you write `#<issue_number>` it links the issue for you) - we know the number of parameters will (almost definitely) correlate with fine-tuned performance, we want to tease out whether the base PARC metric adds anything independently of that.

**Comment by eddableheath** [comment]
Closed as duplicated by #20

**Comment by eddableheath** [comment]
Closed as duplicated by https://github.com/alan-turing-institute/ARC-LoCoMoSeT/issues/20
